### ISSUE TITLE
[BUG] Unable to toggle between `Checkpoints` and `Rules` in `Accessibility Checker Side Panel` when width too small

### ISSUE THREAD

**Issue by fbarroso24** [IBMa/equal-access#263]
## Title line template: [Title]: Brief description
When the `Accessibility Checker Dev Tools Side Panel` is too small, on my screen it was less than `672px` I lost the ability to view `Violations` by `Checkpoints` or `Rules`.  

## What tool(s) are you using?
- [ ] `accessibility-checker-engine`
- [x ] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`

## Detailed description

When the `Accessibility Checker Dev Tools Side Panel` is too small, on my screen it was less than `672px` I lost the ability to view `Violations` by `Checkpoints` or `Rules`.  
![image](https://user-images.githubusercontent.com/8657768/97438206-dd61d900-18fa-11eb-82ef-5ed3fb36805b.png)

Only by luck did I discover that if I increased the width of the `Accessibility Checker Side Panel` that only then would these options become available.  Here's a screenshot showing the panel in question.
![image](https://user-images.githubusercontent.com/8657768/97439263-531a7480-18fc-11eb-87f1-0f1b79001557.png)

## Steps to reproduce the issue
1. Open up Chrome `Dev Tools` and dock it to the right
2. Open up the `Accessibility Checker` in the dev tools side panel
3. Slowly start to decrease the width of the panel until you notice the options disappear altogether.

**Comment by drjoho** [comment]
We have a new release in the next couple of weeks and this issue is addressed.
